style.scss

```diff
@@ -104,38 +104,43 @@ body {
         height: fit-content;
         display: flex;
         flex-direction: row;
-        justify-content: flex-start;
+        justify-content: space-between;
         overflow-x: scroll;
+        gap: 0.6rem;
         scrollbar-width: none;
+        margin-bottom: 0.5rem;
 
         .city-chip {
-          background-color: transparent;
+          background-color: #6153cc22;
```

Comment:
Opacity in the hex value isn't very popular because the meaning isn't super clear. Just resolve this comment though as Tailwind is coming.